mailpoet/assets/js/src/common/listings/_stories/newsletter_status.tsx

```diff
@@ -2,6 +2,7 @@ import { addHours, subHours } from 'date-fns';
 import { MailPoet } from 'mailpoet';
 import { NewsletterStatus } from '../newsletter_status';
 import { Heading } from '../../typography/heading/heading';
+import { NewsletterStatus as NewsletterStatusEnum } from '../../../newsletters/models';
```

Comment:
I believe this can be replaced with
```suggestion
import { NewsletterStatus as NewsletterStatusEnum } from 'newsletters/models';
```